arithmetization/src/test/java/net/consensys/linea/zktracer/ReplayTests.java

```diff
@@ -553,17 +509,26 @@ void hotOrColdPrecompile() {
 
   // TODO: should be replace by a unit test triggering a CALLDATACOPY in a ROOT context of a
   // deployment transaction
+
   @Test
   void callDataCopyCnNotFound() {
     replay("67050-67059.json.gz");
   }
 
-  /**
```

Comment:
@lorenzogentile404 this test (and comment) shouldn't be deleted

Reply:
I reverted the unnecessary changes in `ReplayTests.java` and just added the relevant test for this issue.